internal/experiment/webconnectivityqa/tcpblocking.go

```diff
@@ -63,6 +63,10 @@ func tcpBlockingConnectionRefusedWithInconsistentDNS() *TestCase {
 		},
 		ExpectErr: false,
 		ExpectTestKeys: &testKeys{
+			BodyLengthMatch:       true,
+			StatusCodeMatch:       true,
+			HeadersMatch:          true,
+			TitleMatch:            true,
```

Comment:
This is not a regression. LTE tries harder and is able to bypass the blocking. But it still records the existing blocking mechanisms. So, we are actually doing a better job here.